multimodal/tests/unittests/others/test_object_detection.py

```diff
@@ -55,38 +92,43 @@ def test_mmdet_object_detection_inference_dict(checkpoint_name):
 @pytest.mark.parametrize(
     "checkpoint_name",
     [
-        "faster_rcnn_r50_fpn_2x_coco",
         "yolov3_mobilenetv2_320_300e_coco",
-        "detr_r50_8x2_150e_coco",
     ],
 )
-def test_mmdet_object_detection_fit_then_evaluate_coco(checkpoint_name):
-    data_dir = download_sample_dataset()
+def test_mmdet_object_detection_inference_list(checkpoint_name):
+    mmdet_image_name = download_sample_images()
 
-    train_path = os.path.join(data_dir, "Annotations", "trainval_cocoformat.json")
-    test_path = os.path.join(data_dir, "Annotations", "test_cocoformat.json")
-    # Init predictor
     predictor = MultiModalPredictor(
         hyperparameters={
             "model.mmdet_image.checkpoint_name": checkpoint_name,
-            "env.num_gpus": 1,
+            "env.num_gpus": 1,  # currently mmdet only support single gpu inference
```

Comment:
I think we support multigpu inference here?

Reply:
right, this was copied from the notebooks. We can add tests for multi-gpu as well.

Reply:
We may not need to but this comment shall be removed as it's misleading.